taskvine/src/manager/vine_manager.c

```diff
@@ -935,6 +935,49 @@ static void cleanup_worker_files(struct vine_manager *q, struct vine_worker_info
 	hash_table_free_keys_array(cachenames);
 }
 
+/** Evict a random worker to simulate a failure. */
+int vine_manager_evict_a_random_worker(struct vine_manager *q)
```

Comment:
Let's all it release_random_worker.  Eviction means to kick off a worker from a compute node.